src/utils.ts

```diff
@@ -8,7 +8,7 @@
  *     http://www.apache.org/licenses/LICENSE-2.0
  */
 
-export type ReactChildren = (React.ReactNode) | (React.ReactNode[] & React.ReactNode);
+export type ReactChildren = React.ReactNode | (React.ReactNode[] & React.ReactNode);
```

Comment:
```suggestion
export type ReactChildren = React.ReactNode[] & React.ReactNode;
```
Doesn't this cover all?